# idk czemu różne
FZ_EST_N: 8.0
FZ_CTRL_N: 2.0

# display message
verbose: false
